src/services/api.py

```
@@ -29,22 +31,274 @@
+            mode_used = "model"
+            stylizer = get_diffusion_stylizer()
+            pix = pixel_art_person_diffusion(image, mask, stylizer)
+            if background == "original":
+                base = image.convert("RGBA")
+                base.paste(pix, (0, 0), pix)
+                out = base
+            else:
+                out = pix
+
+        else:
+            raise ValueError("style must be one of: mosaic, pixelart, model")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)